.github/pr-title-checker-config.json

```diff
@@ -0,0 +1,26 @@
+{
+    "LABEL": {
+        "name": "The PR title has to contain a package name like \"[ffigen]\", or \"[infra]\".",
```

Comment:
This is the label name, and not the description, IIUC. Might want to opt for something shorter.

Reply:
Maybe we should just leave the label away completely. There's already a red/green bot checkmark/cross.